### DISCUSSION TITLE
Printable cheatsheet generated from config?

### DISCUSSION THREAD

**Discussion by scottromack** [tailwindlabs/tailwindcss#4876] [Ideas]
I'm wondering how difficult it would be to generate all classes from the current codebase/config so that something like https://raw.githubusercontent.com/gojutin/tailwind-cheatsheet/main/printable-tailwindcss-cheatsheet.jpg would always be in sync.

Thanks, I adore tailwind.

**Comment by olets** [comment]
Not _exactly_ what you're asking about, but depending on your use case https://github.com/rogden/tailwind-config-viewer might be a solution.

**Reply by scottromack** [reply]
Very cool. Thanks!
